pkgs/by-name/go/gophernicus/package.nix

```
@@ -0,0 +1,80 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  gnused,
+  systemdMinimal,
+  systemdSupport ? lib.meta.availableOn stdenv.hostPlatform systemdMinimal,
+  nix-update-script,
+  nixosTests,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "gophernicus";
+  version = "3.1.1";
+
+  src = fetchFromGitHub {
+    owner = "gophernicus";
+    repo = "gophernicus";
+    tag = finalAttrs.version;
+    hash = "sha256-pweiUiMmLXiyF9NMxvcWfJPH6JiGRlpT4chJiRGh9vg=";
+  };
+
+  postPatch = ''
+    substituteInPlace README.md \
+      --replace-warn 'DEVEL' '${finalAttrs.version}'
+
+    substituteInPlace src/gophernicus.h \
+      --replace-fail 'SAFE_PATH    "/usr/bin:/bin"' 'SAFE_PATH    "/usr/bin:/bin:/run/gophernicus/bin"'
+
+    ${lib.optionalString stdenv.hostPlatform.isDarwin ''
+      ${lib.getExe gnused} -E -i '/^\s+(chown|chmod)/d' Makefile.in
+    ''}
+  '';
```

```suggestion
  '' + lib.optionalString stdenv.hostPlatform.isDarwin ''
    sed -E -i '/^\s+(chown|chmod)/d' Makefile.in
  '';
```
sed is in the stdenv